apps/sms/test/marionette/lib/messages.js

```diff
@@ -87,6 +88,8 @@
       var actions = client.loader.getActions();
 
       return {
+        client: client,
```

Comment:
Just noted I didn't answered to this particular point in the main thread. We need it for [this call](https://github.com/mozilla-b2g/gaia/pull/29631/files#diff-c144504149c41c787f698826512bd561R20)